src/fides/api/ops/api/v1/endpoints/utils.py

```diff
@@ -28,3 +29,18 @@ def validate_start_and_end_filters(
                 status_code=HTTP_400_BAD_REQUEST,
                 detail=f"Value specified for {field_name}_lt: {end} must be after {field_name}_gt: {start}.",
             )
+
+
+def transform_fields(
```

Comment:
@allisonking this is my "magic" function, but now it's causing more bugs 🤷 should be what we need here though, figuring out why it's breaking things

Reply:
that's slick!

Reply:
Oh in the actual func I put a check but messed it up! I'll fix and try again. I think something a bit weird is going on so I'm digging in